### ISSUE TITLE
Can't save headers for chained requests

### ISSUE THREAD

**Issue by oshabal** [taverntesting/tavern#510]
Hi there,
I am trying to do simple sign_in/sign_out test.
After sign_in I need to save few headers to use them in sign_out.
Like that:
```
    save:
      headers:
        access-token: access-token
        client: client
        uid: uid
```
But it doesn't work, I'm keep getting unclear purest warnings about wrong saving of nested data. 
When trying to do a sign_out 'm getting an errors about that variable (e.g {access-token:s} is None.
What is wrong with that?

**Comment by oshabal** [comment]
Also tried like that, no luck
https://stackoverflow.com/questions/58997357/what-does-a-jmes-path-query-look-like-when-saving-a-variable-in-tavern

**Comment by oshabal** [comment]
Solved

**Comment by benhowes** [comment]
@oshabal glad you got it resolved, would you mind posting the solution so we can update the docs to make it clearer?

**Comment by oshabal** [comment]
Sure. I'm using Atom as my IDE, which have YAML linter/verifier plugin, so I suppose that it was the plugin bug, not Tavern.
'{access-token:s}' - this doesn't work
"{access-token:s}" - this works perfectly but I keep getting warnings about that I should save data using JMESpath syntax, which I still can't understand properly. Anyway test is working, data is saved. I believe that it will be fixed in 1.0 release.
